src/main/interface/msp.c

```diff
@@ -1277,8 +1274,38 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU16(dst, currentPidProfile->itermThrottleThreshold);
         sbufWriteU16(dst, currentPidProfile->itermAcceleratorGain);
         sbufWriteU16(dst, currentPidProfile->dtermSetpointWeight);
-        break;
+        sbufWriteU8(dst, currentPidProfile->iterm_rotation);
+#if defined(USE_SMART_FEEDFORWARD)
+        sbufWriteU8(dst, currentPidProfile->smart_feedforward);
+#else
+        sbufWriteU8(dst, 0);
+#endif
+#if defined(USE_ITERM_RELAX)
+        sbufWriteU8(dst, currentPidProfile->iterm_relax);
```

Comment:
If we're going to include `iterm_relax` I think we should also include `iterm_relax_type` and `iterm_relax_cutoff`.

Reply:
Fair enough. I used a list of features to be added that @fujin and myself had identified, and the list of parameters in `src/main/interface/settings.c` to put these together - if stuff for one feature isn't adjacent in there I would have missed it. Will amend.